### ISSUE TITLE
Pleural plaques

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6548]
**Preferred term label:** Pleural plaques

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pleural plaques are deposits of hyalinized collagen fibres in the parietal pleura (preferentially pleura adjacent to ribs, particularly the sixth through ninth ribs). Pleural plaques are indicative of asbestos exposure. PMID: 929215

**Parent term (use hpo.jax.org/app)**
Pleural thickening HP:0031944

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added term
